netbox_topology_views/forms.py

```diff
@@ -461,6 +461,7 @@ class IndividualOptionsForm(NetBoxModelForm):
                 'group_locations',
                 'group_racks',
                 'draw_default_layout',
+                'straight_cables',
```

Comment:
Why did you omit an entry in class DeviceFilterForm?

Reply:
Sorry, don't understand this, could you elaborate where I removed the entry?

Reply:
Nevermind, figured it out. Added.